vhdx/src/vhdx_header.rs

```diff
@@ -0,0 +1,536 @@
+extern crate log;
+
+use byteorder::{ByteOrder, LittleEndian, ReadBytesExt, WriteBytesExt};
```

Comment:
I think you can get rid of the need for this crate by using the equivalent functions from `u32`, `u64` and `u128` Rust primitives.